guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -1,24 +1,79 @@
 [id="preparing-a-host-for-external-databases_{context}"]
 = Preparing a host for external databases
 
-Install a freshly provisioned system with the latest {EL} 8 to host the external databases.
+Install a freshly provisioned system with the latest {EL} 9 or {EL} 8 to host the external databases.
 
 ifdef::satellite[]
 Subscriptions for {RHEL} do not provide the correct service level agreement for using {Project} with external databases.
 You must also attach a {Project} subscription to the base operating system that you want to use for the external databases.
 endif::[]
 
+ifdef::satellite[]
+Use the instructions in {InstallingServerDocURL}attaching-infrastructure-subscription_{project-context}[Attaching the {Project} Infrastructure Subscription] to attach a {Project} subscription to your server.
+endif::[]
+
 .Prerequisites
 * The prepared host must meet {Project}'s {InstallingServerDocURL}storage-requirements_{project-context}[Storage Requirements].
 
 .Procedure
 ifndef::orcharhino[]
-. Use the instructions in {InstallingServerDocURL}attaching-infrastructure-subscription_{project-context}[Attaching the {Project} Infrastructure Subscription] to attach a {Project} subscription to your server.
-. Disable all repositories and enable only the following repositories:
+Select the operating system and version you are installing external database on:
+
+* xref:#repositories-externaldb-el-9[{EL} 9]
+* xref:#repositories-externaldb-el-8[{EL} 8]
+
+[id="repositories-externaldb-el-9"]
+== {EL} 9
+
+:package-manager: dnf
```

Comment:
This attribute appears unused.

Reply:
This will be required for `snip_step-verify-enabled-repolist.adoc`

Reply:
However, we can simply keep `dnf` because that step does not apply to Debian builds.